examples/functions/timetable/Netex_01.1_Bus_SimpleTimetable_JourneysOnly.xml

```diff
@@ -220,6 +220,17 @@ The Calendar is shown coded as
 							<VehicleModes>bus</VehicleModes>
 						</ScheduledStopPoint>
 					</scheduledStopPoints>
+					<occupancies>
+						<OccupancyView version="any" id="occinFrame">
```

Comment:
I don't see how a Occupancy**View** at this point. I think it must not be a _view_.

Reply:
No other elements available. It always was a View and never anything else. You wanted it here. Sorry.

Reply:
No I want to have an _Occupancy_ here.